### PR TITLE
Spell check

### PR DESCRIPTION
**Pull request by pedrouid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,11 +99,11 @@ It is important to note that an update might have an earlier anchor record but n
...
 
 This is essentially a *double spend attack* which is the problem that blockchains solve. However since identities have only one owner, the user, this is less of a problem. In this case, a "double spend" would cause the user to lose all history and associations that have accrued on their identity, which they are naturally disincentivized to do. Similarly for tile documents signed by multiple parties, since all parties need to sign all updates they all would need to all be aware of the data withholding attack.
 
-In the case of organizational identities this is more of a problem, e.g. if an old admin of the org wants to cause trouble. This can be solved by introducing "heavy anchors" which rely more heavily on some on-chain mechanism. For example, a smart contract or a DAO that controls the identity.
+In the case of organizational identities this is more of a problem, e.g., if an old admin of the org wants to cause trouble. This can be solved by introducing "heavy anchors" which rely more heavily on some on-chain mechanism. For example, a smart contract or a DAO that controls the identity.
```
Comment (oed) [inline README.md:106]: This change seems wrong `e.g.,` is strange.

**Comment by pedrouid** [comment]
Not using any specific tool, I just noticed some typos and copied the Markdown to [Typora](https://typora.io/) then checked one by one manually.

**Comment by pedrouid** [comment]
You could actually open the whole repository on Typora and edit files from there actually

**Comment by pedrouid** [comment]
Pretty nice experience

Screenshot 2020-04-07 14 58 39

**Pull request merged**